### PR TITLE
 cosbench.py: do not compose "nodes" using user@host 

### PR DESCRIPTION
**Pull request by tchaikov**

### CODE REVIEW

File: benchmark/cosbench.py [hunk 1]
```diff
@@ -279,7 +279,7 @@ def check_cosbench_res_dir(self):
 
     def _run(self):
         conf = self.config
-        stdout, stderr = common.pdsh("%s@%s" % (self.user, conf["controller"]),'sh %s/cli.sh submit %s/%s.xml' % (conf["cosbench_dir"], conf["cosbench_xml_dir"], conf["xml_name"])).communicate()
+        stdout, stderr = common.pdsh(conf["controller"], 'sh %s/cli.sh submit %s/%s.xml' % (conf["cosbench_dir"], conf["cosbench_xml_dir"], conf["xml_name"])).communicate()
```
Comment (rzarzynski) [inline benchmark/cosbench.py]: In the constructor we still do:

```python
    self.user = settings.cluster.get('user')
```

which might be unnecessary after the changes. The `user` key is also present in some example files (like `example/wip-cosbench/cosbench_ex.yaml`).

In the ceph repo we use this setting (`user: 'ubuntu'`) in:
```
qa/suites/rados/perf/workloads/cosbench_64K_write.yaml
qa/suites/rados/perf/workloads/cosbench_64K_read_write.yaml
qa/suites/perf-basic/workloads/cosbench_64K_write.yaml
```

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2019-08-13_07:59:21-perf-basic-wip-kefu-testing-2019-08-12-1306-distro-basic-mira/

File: benchmark/cosbench.py (hunk 1 above)
Comment (tchaikov) [reply benchmark/cosbench.py]: @rzarzynski removed from `Cosbench`.

`user: 'ubuntu'` is used. but i am not sure what you suggest. what i can tell is that it's still needed in cosbench tests. as we might want to use it when deploying ceph cluster if it's a use_existing cluster or a local run.

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2019-08-13_08:53:40-perf-basic-wip-kefu-testing-2019-08-12-1306-distro-basic-mira/

**Comment by rzarzynski** [comment]
http://pulpito.front.sepia.ceph.com/rzarzynski-2019-08-13_10:44:12-perf-basic-wip-kefu-testing-2019-08-12-1306-distro-basic-mira/

**Pull request merged**